templates/analytics/sponsorship_forms_support.html

```diff
@@ -22,6 +22,7 @@
 
 <form method="POST" class="remote-form">
     <b>Discount</b>:<br />
+    <i>Updates will change pre-existing plans and scheduled upgrades.</i><br />
```

Comment:
Do we want to clarify exactly how this works? E.g. I think it only will affect future charges -- existing prepaid will be fine.

Reply:
Sure. I added:

> **Discount percentage**:
> *Updates will change pre-existing plans and scheduled upgrades.*
> *Any prorated licenses for the current billing cycle will be billed at the updated discounted rate.*

So, prorated licenses added during a billing cycle are not billed until the next cycle's renewal payment. And they would all be at the new discounted rate in a future bill.

Feel free to edit that text though for more clarity.